packages/DataGrid/columns/number/DataGridColumnCurrency.ts

```diff
@@ -13,16 +13,24 @@ export class DataGridColumnCurrency<TData> extends DataGridColumnNumberBase<TDat
 	static defaultCurrency?: Currency
 
 	@property({ type: Object, converter: FieldCurrency.currencyConverter }) currency = DataGridColumnCurrency.defaultCurrency
+	@property() currencyDataSelector?: string
```

Comment:
- A data-selector should be of type `KeyPathOf<TData>` I think (see how it is defined in other column components)